src/oc_erchef/apps/oc_chef_wm/src/chef_wm_status.erl

```diff
@@ -36,6 +36,7 @@
 
 -include_lib("webmachine/include/webmachine.hrl").
 -define(A2B(X), erlang:atom_to_binary(X, utf8)).
+-define(VERSION_PATH, '/opt/opscode/version-manifest.txt').
```

Comment:
I believe we have a JSON version of this file that might be a bit more consistent to parse.

Reply:
will take a look at it